js/signin.js

```diff
@@ -0,0 +1,60 @@
+function checkEmail() {
+    const val = document.getElementById("email-login").value;
+    var re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
What are you testing here?

Reply:
I am using regular expression to test whether the email is valid